copier/cli.py

```diff
@@ -189,6 +191,17 @@ def data_switch(self, values: List[str]) -> None:
             values: The list of values to apply.
                 Each value in the list is of the following form: `NAME=VALUE`.
         """
+        files = [e for e in values if "=" not in e]
+        values = [e for e in values if "=" in e]
+        print(files)
+        print(values)
+        for file in files:
+            if not Path(file).exists():
+                print(colors.red | f"File: {file} doesn't exist", file=sys.stderr)
+            if not re.match(r".+\.ya?ml$", file, re.IGNORECASE):
```

Comment:
I find it more ergonomic to avoid regexp and use `Path(file).suffix not in {"yaml", "yml"}`